### ISSUE TITLE
Arduino nano / mcp2515

### ISSUE THREAD

**Issue by ghira89** [collin80/SavvyCAN#625]
I'm looking for help for Arduino nano and mcp2515, one already canhacker but I can't make it work on savvycan..

**Comment by amiralnar** [comment]
Arduino must be configured in hardware flow control mode of COM port.

**Comment by ghira89** [comment]
Yes read message but not send.. and no set speed can.. using terminal 1.9 for set speed

**Comment by amiralnar** [comment]
That's what i've got too. 
I was able to set speed and open CAN connection using terminal and then listen CAN messages.
To fix this issue i had to build my fork with some changes in the source code.
This changes is incompatible with original GVRET adapter, so you may try it with Arduino
https://github.com/amiralnar/SavvyCAN/releases

**Comment by ghira89** [comment]
Test program but speed not work.. select speed on terminal 1.9

**Comment by amiralnar** [comment]
If you mean bus speed then you have to set it in next window after connection established

**Comment by amiralnar** [comment]
And then after some data the connection may be lost. Arduino is very unstable.

**Comment by ghira89** [comment]
Work but not possible speed bus select
